Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -82,7 +94,11 @@ sudo usermod -a -G dialout $USER
 if dpkg-query -l "modemmanager"; then
     $APT_GET remove modemmanager
 fi
+
 $APT_GET update
+if $IS_WINDOWS_10_WSL; then
+	$APT_GET upgrade
```

Comment:
Does this need a `-y`?

Reply:
not sure, probably as likely as the update would. Why?